plugins/quay/src/components/QuayDashboardPage/QuayDashboardPage.tsx

```diff
@@ -1,21 +1,6 @@
-import { useEntity } from '@backstage/plugin-catalog-react';
 import React from 'react';
 import { QuayRepository } from '../QuayRepository';
-import { useQuayAppData } from '../useQuayAppData';
 
 export const QuayDashboardPage = () => {
-  const { entity } = useEntity();
-  const { repositorySlug } = useQuayAppData({ entity });
-  const info = repositorySlug.split('/');
-
-  const organization = info.shift() as 'string';
-  const repository = info.join('/');
-
-  return (
-    <QuayRepository
-      organization={organization}
-      repository={repository}
-      widget={false}
-    />
-  );
+  return <QuayRepository />;
```

Comment:
Thinking about this.. do we need a new component that just renders a single component - basically just reexports it? What about we change the router line (https://github.com/janus-idp/backstage-plugins/pull/240/files#diff-d2fefa7438680cfd44aad206e463e9058a17ea3282f2ca483ea4033a41497806R22 ) to use the `QuayRepository` directly? WDYT?